packages/experimental/mosaic/src/layout/layout.tsx

```diff
@@ -31,7 +31,7 @@ export interface Layout {
 export type Item<T extends {} = {}> = {
   id: string;
   data?: T;
-  label: string;
+  title: string;
```

Comment:
I've always waffled around between `label` and `title` personally… here’s what I know with some certainty:

- Objects with `label` should have an optional `description`, these are first-class UI concepts that go together
- `label` is a UI idea rather than a kind of user content
- Objects with `title` could also have `body` for the main text content and `abstract` which summarizes the body (can be derived directly from `body` or not)
- `title` is a kind of user content rather than part of the software (with certain rare exceptions)

Reply:
Ah nice.
I was in a slightly different place -- but have also been switching back and forth.
I think I prefer title and body as UI concepts since they shadow concepts in HTML; this way there are less likely to be confused with user data structures. BUT, as I think we've been in agreement from the outset, components should define their own data semantics and either provide accessors or some more advanced "model" interface that clients resolve.